lunchbot.py

```diff
@@ -154,13 +167,40 @@ def handle_lunchbot_command(self, command, user):
                 self.update_data()
                 self.read_data()
             else:
-                post_ephemeral = True
+                self.post_ephemeral = True
                 msg = 'Uh oh! %s is not in the list of restaurants.' % restaurant
+        except Exception as e:
+            self.post_ephemeral = True
+            msg = 'Please try again.\n%s' % e
+        return msg
+
+    def handle_lunchbot_command(self, command, user):
+        """
+        Takes a command from Slack chat directed at Lunchbot and performs the action if it is supported
+        """
+        self.post_ephemeral = False
+
+        if 'help' in command.lower():
+            self.post_ephemeral = True
+            msg = self.HELP_TEXT
+        elif 'list restaurants' in command.lower():
+            self.post_ephemeral = True
+            msg = self.BLANK
```

Comment:
I can't find self.BLANK anywhere in this file anymore, did you remove it completely?

Reply:
>_< added it back in